modules/db/src/test/java/org/mule/module/db/internal/domain/type/LobResolvedDataTypeTestCase.java

```diff
@@ -29,34 +32,60 @@
 import org.junit.rules.ExpectedException;
 
 @SmallTest
-public class ClobResolvedDataTypeTestCase extends AbstractMuleTestCase {
+public class LobResolvedDataTypeTestCase extends AbstractMuleTestCase {
```

Comment:
Used separated classes to test clboa and blob types. If you need to reuse, then create an abstract superclass